src/test/groovy/graphql/schema/idl/SchemaPrinterTest.groovy

```diff
@@ -1892,4 +1892,19 @@ type MyQuery {
 """
     }
 
+    def "allow printing of just directives"() {
+        def sdl = """
+            directive @foo on FIELD_DEFINITION
+            type Query { anything: String @foo }
+        """
+        def schema = TestUtil.schema(sdl)
+        def directive = schema.getDirective("foo");
+
+        when:
+        def result = new SchemaPrinter(defaultOptions().includeDirectives(true)).print(directive)
+
+        then:
+        result == """directive @foo on FIELD_DEFINITION"""
+    }
+
```

Comment:
If we come in through this "direct method" so we want to require the side effect of `includeDirectives(true)` ?